napari/layers/image/image.py

```diff
@@ -689,3 +689,52 @@ def contrast_limits(self, contrast_limits):
                 self.refresh(highlight=False, extent=False)
             finally:
                 self._keep_auto_contrast = prev
+
+    def _calculate_value_from_ray(self, values):
+        # translucent is special: just return the first value, no matter what
+        if self.rendering == ImageRendering.TRANSLUCENT:
+            return np.ravel(values)[0]
```

Comment:
Why this will not return 0?

Reply:
Why would it?

Reply:
Because we select the first point of the list, so why is it non-zero?

Reply:
translucent rendering returns the first element, even if it is 0. 

<img width="1353" alt="Screenshot 2024-09-24 at 10 06 58 PM" src="https://github.com/user-attachments/assets/82941421-febe-4b1e-8096-31be9676df07">